lib/CXGN/Stock/ParseUpload/Plugin/AccessionsGeneric.pm

```diff
@@ -104,6 +104,18 @@ sub _validate_with_plugin {
         $errors{'missing_species'} = \@species_missing;
     }
 
+    # Check for existing non-accession stocks
+    my $accession_type_id = SGN::Model::Cvterm->get_cvterm_row($schema, 'accession', 'stock_type')->cvterm_id();
+    my $accession_list = $parsed_values->{'accession_name'};
+    my $stocks_in_db_rs = $schema->resultset("Stock::Stock")->search({ uniquename => { -ilike => $accession_list }, type_id => { '<>' => $accession_type_id } });
```

Comment:
shouldn't accessions be included as well?

Reply:
No, the the accession upload can handle existing accessions - it updates them with any new properties.